### ISSUE TITLE
Can the view setting be changed to override the preference

### ISSUE THREAD

**Issue by peterprib** [archi-contribs/specialization-plugin#10]
Not sure why on a view the setting can be overridden once set in preferences

**Comment by herve91** [comment]
Hi Peter,

I'm not sure what is your question here.

Basically, the preferences allow to indicate when the specialization plugin should replace labels and icons : always, never or depending on the views properties.

Best regards
Hervé

**Comment by peterprib** [comment]
May be better explained by saying preferences should be default which can be overridden at view level.  That is I want labels and in certain views I want to set it to another value

**Comment by herve91** [comment]
I see ... You wish to specify, view per view, the name of the property that will be used as the label.

This is already on my todo list.

**Comment by peterprib** [comment]
Correcto.  Better still if it was per type and at preference level as well.
